src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -411,4 +414,64 @@
                 .status(deleteOrganisationResponse.getStatusCode())
                 .body(deleteOrganisationResponse);
     }
+
+    @ApiOperation(
+            value = "Update the MFA preference of an Organisation",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "The MFA preference of the organisation has been successfully updated"
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request was provided"
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = "Unauthorized Error : The requested resource is restricted and requires authentication"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = "Forbidden Error: Access denied"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "No Organisation was found with the given organisationIdentifier"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
+            )
+    })
+    @PutMapping(
+            path = "/{orgId}/mfa",
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured("prd-admin")
+    public ResponseEntity<Object> updateOrgMfaStatus(@Valid @NotNull @RequestBody MfaUpdateRequest mfaUpdateRequest,
+                                                     @Pattern(regexp = ORGANISATION_IDENTIFIER_FORMAT_REGEX,
+                                                     message = ORG_ID_VALIDATION_ERROR_MESSAGE)
+                                                     @PathVariable("orgId") @NotBlank String organisationIdentifier) {
+
+        log.info("{}:: Received request to update organisation mfa preference...", loggingComponentName);
+
+        organisationIdentifierValidatorImpl.validateOrganisationExistsWithGivenOrgId(organisationIdentifier);
+
+        Organisation organisation = organisationService.getOrganisationByOrgIdentifier(organisationIdentifier);
+
+        if (isNotTrue(organisation.isOrganisationStatusActive())) {
+            throw new InvalidRequest("The requested Organisation is not 'Active'");
+        }
+
+        mfaStatusService.updateOrgMfaStatus(mfaUpdateRequest,organisation);
+
+        return ResponseEntity
+                .status(200)

```

Comment:
I think we should not return `200` here but instead from the `mfaStatusService` method. Otherwise we will always return `200` success when it may not be successful - we had similar issue to another API before